include/CL/cl2.hpp

```diff
@@ -1803,7 +1803,7 @@ class Wrapper
 
     cl_type& operator ()() { return object_; }
 
-    const cl_type get() const { return object_; }
+    cl_type get() const { return object_; }
```

Comment:
I think this is OK - since the wrapper class wraps OpenCL handles it's always OK to return a copy of the handle by value even when the wrapper object is "const".  After this change though, do we need both this function and the non-const overload below, or can the non-const overload be removed?

Reply:
Good catch, I can't think of a good reason to keep the non-const overload, I'll remove it.